noisemodelling-pathfinder/src/main/java/org/noise_planet/noisemodelling/pathfinder/utils/MirrorReflection2D.java

```diff
@@ -0,0 +1,156 @@
+package org.noise_planet.noisemodelling.pathfinder.utils;
+
+import org.locationtech.jts.geom.Coordinate;
+
+public class MirrorReflection2D {
```

Comment:
is class MirrorReflection2D used in noisemodelling ? there is french words in it, weird